puppet/modules/web/manifests/init.pp

```diff
@@ -68,6 +68,7 @@
     docroot_group   => 'website',
     docroot_mode    => '0755',
     custom_fragment => template('web/web.conf.erb'),
+    default_vhost   => true,
```

Comment:
Is this used and does it have an effect? I thought it defined a mapping above to serve it via plain HTTP.

Reply:
hrm, yes and no... this was a brainfart, but the original problem is still unsolved then: LE is putting the response into /var/www/html, which is not served